src/main/java/ome/services/fulltext/FullTextBridge.java

```diff
@@ -245,6 +248,41 @@ public void set_annotations(final String name, final IObject object,
         }
     }
 
+    /**
+     * Walks the acquisition related metadata including channel names.
+     *
+     * @param name
+     * @param object
+     * @param document
+     * @param opts
+     */
+    public void set_acquisition(final String name, final IObject object,
+                                final Document document, final LuceneOptions opts) {
+        if (object instanceof Image) {
+            final Image image = (Image) object;
+            if (image.sizeOfPixels() == 0) {
+                return;
+            }
+            final Pixels pixels = image.getPrimaryPixels();
+            if (pixels == null) {
+                return;
+            }
+            final Iterator<Channel> channelIterator = pixels.iterateChannels();
+            while (channelIterator.hasNext()) {
+                final Channel channel = channelIterator.next();
+                final LogicalChannel logical = channel.getLogicalChannel();
```

Comment:
From today's `Indexer-0.log`,
```
Caused by: java.lang.NullPointerException: null
	at ome.services.fulltext.FullTextBridge.set_acquisition(FullTextBridge.java:273) ~[omero-server.jar:5.5.3-SNAPSHOT]
	at ome.services.fulltext.FullTextBridge.set(FullTextBridge.java:133) ~[omero-server.jar:5.5.3-SNAPSHOT]
	at ome.util.DetailsFieldBridge.set(DetailsFieldBridge.java:46) ~[omero-model.jar:5.5.2-SNAPSHOT]
```

Reply:
Grr... ok. I'll fix the immediate bug, but are you looking at the wider fix or shall I?

Reply:
```
                final LogicalChannel logical = channel.getLogicalChannel();
```
Who'd have thought!

Reply:
I hadn't done much beyond looking at the code enough to conclude that it's not as simple as I'd first hoped but happy to prioritize it. (It shouldn't take more than a couple of days, can put it above downloader, etc.)

Reply:
Should I try to just put something in `set()` itself?

Reply:
Good idea if you don't mind that interim fix. I was going to look at the Google Suite stuff before coming back to the wider fix (though can do otherwise).

Actually I could open a PR to simply make indexer ignore NPEs while not being the fix I wish it had been, if you like; I can come back to the nicer fix later.